sys-power/powerstat/powerstat-0.02.11.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Laptop power measuring tool"
+HOMEPAGE="https://launchpad.net/ubuntu/+source/${PN}"
+SRC_URI="http://kernel.ubuntu.com/~cking/tarballs/${PN}/${P}.tar.gz"
```

Comment:
This can be added in `SRC_URI` as a backup too: https://github.com/ColinIanKing/powerstat/archive/V0.02.11.tar.gz